barretenberg/cpp/src/barretenberg/dsl/acir_format/honk_recursion_constraint.test.cpp

```diff
@@ -172,7 +172,7 @@ class AcirHonkRecursionConstraint : public ::testing::Test {
 
         AcirFormat constraint_system{};
         constraint_system.varnum = static_cast<uint32_t>(witness.size());
-        constraint_system.recursive = false;
+        constraint_system.recursive = true;
```

Comment:
set this to be true because it's used as an inner circuit in a couple tests. This might matter down the line if we ever use .is_recursive_circuit for our transcript hash function